test/setup.js

```diff
@@ -1 +1,21 @@
 import 'babel-polyfill';
+import path from 'path';
+import i18n from 'i18n';
+import clientManager from '../src/ffi/client_manager';
+import ffiLoader from '../src/ffi/ffi_loader';
+
+i18n.configure({
+  locales: ['en'],
+  directory: path.resolve(__dirname, '../', 'locales'),
+  objectNotation: true
+});
+
+i18n.setLocale('en');
+
+// load ffi library
+ffiLoader.loadLibrary()
+// create unregistered client
+  .then(() => clientManager.createUnregisteredClient())
```

Comment:
May be we can move this to test case itself.
```
ffiLoader.loadLibrary()
.catch(err => throw err);
```
Creating the client should be in the test cases and not here